### PR TITLE
SVG background to the Section ~ hero-header-container

### PR DESCRIPTION
**Pull request by PKM777**
## Minor Enhancement

Added a simple SVG background to hero-header-container

## Screenshots

design0
design1

### CODE REVIEW

**Comment by ArslanYM** [comment]
@PKM777  looks cool!

**Comment by Chandraprakash-Darji** [comment]
@PKM777 When ever is there any idea, Just dicuss with the team first

**Comment by Chandraprakash-Darji** [comment]
@AnuragThePathak How it is?

File: src/components/home/Hero/index.module.css [hunk 1]
```diff
@@ -4,6 +4,10 @@
   align-items: center;
   justify-content: center;
   text-align: center;
+  background-image: url('../../../assets/images/chaos.svg') !important;
```
Comment (Chandraprakash-Darji) [inline src/components/home/Hero/index.module.css:7]: don't use imporatant

**Review by Chandraprakash-Darji — CHANGES_REQUESTED** [review]

**Comment by AnuragThePathak** [comment]
Fair enough ig.

File: src/components/home/Hero/index.module.css (hunk 1 above)
Comment (PKM777) [reply src/components/home/Hero/index.module.css:7]: it doesn't work without !important ! :( do you know why ?

Comment (Chandraprakash-Darji) [reply src/components/home/Hero/index.module.css:7]: @PKM777 add to other pages also

**Comment by Chandraprakash-Darji** [comment]
@PKM777 remove important and just increase specificity

CSS of hero section
```css
body .section.header-container {
  height: 100vh;
  display: flex;
  align-items: center;
  justify-content: center;
  text-align: center;
  background-image: url('../../../assets/images/chaos.svg');
  background-repeat: no-repeat;
  background-position: center;
  background-size: cover;
}
```

JSX for the Component
```jsx
  <section className={clsx(styles.section, styles['header-container'])}>
 ```
  update first line

**Comment by PKM777** [comment]
> @PKM777 remove important and just increase specificity
> 
> CSS of hero section
> 
> ```css
> body .section.header-container {
>   height: 100vh;
>   display: flex;
>   align-items: center;
>   justify-content: center;
>   text-align: center;
>   background-image: url('../../../assets/images/chaos.svg');
>   background-repeat: no-repeat;
>   background-position: center;
>   background-size: cover;
> }
> ```
> 
> JSX for the Component
> 
> ```js
>   <section className={clsx(styles.section, styles['header-container'])}>
> ```
> 



> @PKM777 remove important and just increase specificity
> 
> CSS of hero section
> 
> ```css
> body .section.header-container {
>   height: 100vh;
>   display: flex;
>   align-items: center;
>   justify-content: center;
>   text-align: center;
>   background-image: url('../../../assets/images/chaos.svg');
>   background-repeat: no-repeat;
>   background-position: center;
>   background-size: cover;
> }
> ```
> 
> JSX for the Component
> 
> ```js
>   <section className={clsx(styles.section, styles['header-container'])}>
> ```
> 
> update first line

this doesn't work

**Comment by Chandraprakash-Darji** [comment]
@PKM777 have you updated the JSX file?

**Comment by Chandraprakash-Darji** [comment]
@PKM777 is it working?

**Comment by AnuragThePathak** [comment]
@PKM777 are you still working?

**Comment by AnuragThePathak** [comment]
Closing due to inactivity.

**Pull request closed without merging**